docs/_tutorials/spring-cloud-autoconf-jndi.md

```diff
@@ -0,0 +1,415 @@
+---
+layout: tutorials
+title: Spring Cloud Auto-Config JNDI
+summary: A Simple Spring Cloud Application showing how to consume Solace Messaging as a Service provided by Solace Spring JMS Auto-Configuration using JNDI.
+icon: spring-cloud.png
```

Comment:
Need icons that are unique for each tutorial. We have a graphics designer who can help you if needed.